variables.tf

```diff
@@ -116,6 +116,12 @@ variable "manage_reserved_ips" {
   default     = false
 }
 
+variable "number_of_secondary_reserved_ips" {
+  description = "The number of secondary reversed IPs to attach to a Virtual Network Interface (VNI). Additional IPs are created only if `manage_reserved_ips` is set to true."
```

Comment:
I don't think this should rely on manage_reserved_ips, as a user may just want a number of secondary but do not care if terraform is managing them. I can explain further how all that works if needed.